### PR TITLE
Add bulk comment action DAH-588

### PR DESCRIPTION
**Pull request by software-project**
Resolves [DAH-588](https://sfgovdt.jira.com/browse/DAH-588)

### CODE REVIEW

**Comment by akegan** [comment]
@software-project it looks like when you bulk add a status that the substatus isn't being preserved. Status stays the same but substatus disappears in the table + in the popover status history

**Comment by software-project** [comment]
@akegan checking

**Review by akegan — APPROVED** [review]
QA looks good now. Only thing I'm wondering is if it could use another test or 2. Something to catch the issue where the substatus wasn't coming through and getting updated?

File: spec/javascript/e2e/LeaseUpApplicationsPage/status_update.e2e.js [hunk 1]
```diff
@@ -113,6 +113,17 @@ describe('LeaseUpApplicationsPage status update', () => {
         thirdRowUpdatedStatus = await sharedSteps.getText(page, nthRowStatusDropdown(3))
         expect(thirdRowUpdatedStatus).toBe(APPEALED)
 
+        await testBrowser.close()
```
Comment (akegan) [inline spec/javascript/e2e/LeaseUpApplicationsPage/status_update.e2e.js]: I think if you close the browser it'll cause issues in the next test

**Pull request merged**